meson.build

```diff
@@ -0,0 +1,591 @@
+# Maintainer: Perry Werneck <perry.werneck@gmail.com>
+
+# SPDX-License-Identifier: LGPL-3.0-or-later 
+
+# Copyright (C) 2024 Banco do Brasil S. A.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+project(
+	'lib3270', 
+	['c'],
+	version: '5.5.0',
+	license: 'GPL-3.0-or-later',
+)
+
+project_description = 'TN3270 client library'
+product_name = 'pw3270'
+product_domain = 'br.com.bb.pw3270'
+product_title = 'TN3270 Access library'
+
+lib_src = []
+
+#
+# Versioning
+#
+pkg_version = meson.project_version()
+version_array = pkg_version.split('.')
+pkg_major_version = version_array[0].to_int()
+pkg_minor_version = version_array[1].to_int()
+pkg_micro_version = version_array[2].to_int()
+
+libversion = '@0@.@1@'.format(pkg_major_version,pkg_minor_version)
+
+#
+# i18n
+#
+subdir('po')
+
+#
+# Dependencies
+#
+
+c = meson.get_compiler('c')
+libssl = dependency('libssl', required: false)
+libintl = c.find_library('intl', required : false)
+
+lib_extra = [
+  libintl,  
+]
```

Comment:
Since meson 0.59, you can use `dependency('intl')` instead, which is safer and more reliable: https://mesonbuild.com/Dependencies.html#intl